DeviceManager/DeviceManager/View-Model/GoogleSignIn.swift

```diff
@@ -0,0 +1,42 @@
+import UIKit
+import Foundation
+import Firebase
+import GoogleSignIn
+
+class GoogleSignin :NSObject ,GIDSignInDelegate {
+    
+    static let shared = GoogleSignin()
+    let loginvm = LoginVM()
```

Comment:
login vm should not be initiated here, instead login vm should call google handler and observe callback

Reply:
This has not been addressed @Karthik-UK